service-node/src/main/scala/org/enterprisedlt/fabric/service/node/Util.scala

```diff
@@ -265,6 +266,8 @@ object Util {
     def futureDate(shift: Period): Date = Date.from(LocalDate.now().plus(shift).atStartOfDay(ZoneOffset.UTC).toInstant)
 
     def parsePeriod(periodString: String): Period = Period.parse(periodString)
+
+    def getCCLangType(lang: String): Type = Type.values().find(_.toString.contentEquals(lang.toUpperCase)).getOrElse(Type.JAVA)
```

Comment:
Here is a more convenient way for enums:
`Try(Type.valueOf(lang.toUpperCase)).toOption.getOrElse(Type.JAVA)`